docs/src/modules/branding/CustomerIcons.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Container, Typography, Grid, Box } from '@material-ui/core';
+
+const customerIcons = [
+  { image: '/static/branding/pricing/coursera.svg' },
+  { image: '/static/branding/pricing/amazon.svg' },
+  { image: '/static/branding/pricing/nasa.svg' },
+  { image: '/static/branding/pricing/netflix.svg' },
+  { image: '/static/branding/pricing/unity.svg' },
+  { image: '/static/branding/pricing/shutterstock.svg' },
+];
+
+export default function CustomerIcons() {
```

Comment:
To complete replace. This was already implemented on the pricing page (and better, e.g. no layout when the images are loading). Best to abstract what we have on the pricing page.